doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -116,6 +119,15 @@ Matter
     The generated measurements simulate local temperature changes.
     Additionally, you can enable periodic outdoor temperature measurements by binding the thermostat with an external temperature sensor device.
 
+  * Migration of the Node Operational Key Pair (NOK) from the Mbed TLS-related settings storage to the PSA ITS secure storage.
+    All existing NOKs for all Matter fabrics will be migrated to the PSA ITS secure storage at boot.
+
+    If the migration fails, the ``kOperationalKeystoreMigrationFailed`` public event will be emitted and dispatched in the Matter event handler.
+    The default reaction to migration failure is a factory reset of the device.
+    To change the default reaction, set the :kconfig:option:`CONFIG_NCS_SAMPLE_MATTER_FACTORY_RESET_ON_KEY_MIGRATION_FAILURE` Kconfig option to ``n`` and implement the reaction in your Matter event handler.
+    For more information about the Matter event handler, see the :file:`matter_event_handler.h` header file.
+    After the migration, Mbed TLS-related entries in the settings storage will be removed and no longer available.
```

Comment:
Line 129 should probably be moved to be after Line 123, as Lines 125 - 128 concern migration failure. Remember to keep the empty line to maintain the paragraph break.